README.md

```diff
@@ -46,7 +56,132 @@ or
 
 `java -jar <location of the jar file>`
 
+## Endpoints
```

Comment:
Great that you added missing documentation ❤️

Reply:
Thanks. Every time I need to check this project I find myself opening the controller.